fugue/_utils/interfaceless.py

```diff
@@ -431,6 +433,20 @@ def count(self, df: pd.DataFrame) -> int:
         return df.shape[0]
 
 
+class _IterablePandasParam(_DataFrameParamBase):
+    def __init__(self, param: Optional[inspect.Parameter]):
+        super().__init__(param, "Iterable[pd.DataFrame]", "q")
+
+    def to_input_data(self, df: DataFrame) -> Iterable[pd.DataFrame]:
+        return iter(df.as_pandas())
```

Comment:
We should throw exception here, iterable of pd.dataframe should not be used as input.

Also the test didn't cover the input case, maybe we need to add a test to assert it throws